index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/38b6d4ab77.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Zunair Mobile Setup Website</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <nav class="nav-bar">
+          <h3 class="logo">Zunair khan</h3>
+          <i class="fa fa-bars menu-bar" aria-hidden="true"></i>
+        </nav>
+      </header>
+      <div class="headline-section">
+        <div class="headline-container">
+          <div class="heading-1">
+            <h1>Hey There. I'm Zunair</h1>
+          </div>
+          <div class="heading-2">
+            <h2>I'am a software developer</h2>
+          </div>
+          <div class="paragraph">
+            <p>
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded,don’t hestiate to contact me.
+            </p>
+          </div>
+          <div class="social-icons-container">
+            <i class="fa fa-twitter social-icons" aria-hidden="true"></i>
+            <i class="fa fa-linkedin social-icons" aria-hidden="true"></i>
+            <i class="fab fa-medium-m social-icons"></i>
+            <i class="fa fa-angellist social-icons" aria-hidden="true"></i>
+            <i class="fa fa-github social-icons" aria-hidden="true"></i>
+          </div>
```

Comment:
Firstly, I have used icons instead of images so no need to use ul to list items, and as u said using <a> tags so they have been added. Also, the design of icons and their alignment has been corrected as said by you.